index.html

```diff
@@ -3,21 +3,237 @@
+                    <p>Youth involement and Participation among other wards and councils </p>
+                </div>
+                <div class="d-flex my-1 event-bg justify-content-around justify-content-md-center align-items-center flex-md-column col-12 text-center">
+                    <img class="conta" src="img/women.png" alt="Women-involvement">
+                    <h3 class="hello-textr">Women</h3>
+                    <p>Opportunities for Women to help them pay for maternity and other expenses</p>
+                </div>
+            </div>
+            <div class="see-height d-flex justify-content-center align-items-center">
+                <button class="d-lg-none bg-join">
+                    Join Kadoma Community Development
+                </button>
+                <p class="under-p d-none d-lg-block">SEE THE WHOLE SCHEDULE</p>
+            </div>
+        </section>
+    </div>
+
+    <div class="container">
+        <section class="speakers">
+            <div class="my-5">
+                <h3 class="text-center">Featured Speakers</h3>
+                <div class="underline">
+                    <hr>
+                </div>
+            </div>
+            <div class="row">
+                <section class="col-md-6 d-flex f-speaker mb-5">
+                    <div class="img">
+
+                    </div>
+                    <div class="d-flex flex-column ps-3">
+                        <h3></h3>
+                        <p></p>
+                        <p></p>
+                    </div>
+                </section>
+                <section class="col-md-6 d-flex f-speaker mb-5">
+                    <div class="img">
+
+                    </div>
+                    <div class="d-flex flex-column ps-3">
+                        <h3></h3>
+                        <p></p>
+                        <p></p>
+                    </div>
+                </section>
+                <section class="col-md-6 d-flex f-speaker mb-5">
+                    <div class="img">
+
+                    </div>
+                    <div class="d-flex flex-column ps-3">
+                        <h3></h3>
+                        <p></p>
+                        <p></p>
+
+                    </div>
+
+                </section>
+                <section class="col-md-6 d-flex f-speaker mb-5">
+                    <div class="img">
+
+                    </div>
+                    <div class="d-flex flex-column ps-3">
+                        <h3></h3>
+                        <p></p>
+                        <p></p>
+                    </div>
+                </section>
+            </div>
+        </section>
+    </div>
+    <footer>
+        <div class="container-fluid footer-bg">
+            <section class="container d-flex flex-column justify-content-center align-items-center">
+                <div class="partner-header d-flex flex-column justify-content-center align-items-center">
+                    <h4>Partner</h4>
```

Comment:
* Kindly fix this section try to make it look as close as possible to the original design 
* * YOURS : 
![image](https://user-images.githubusercontent.com/52497246/133554112-2d01a7a2-f6e1-4eb2-acb5-a30488de8140.png)
* ORIGINAL : 
![image](https://user-images.githubusercontent.com/52497246/133554164-2c536bf6-2af8-4477-a0e3-510744b4c7a6.png)